src/components/NFTBook/SpecTableItemAvailableFormat.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <component :is="tag">
     <NFTBookSpecTableLabel :text="$t('nft_details_page_label_content_types')" />
-    <div class="font-[600]">{{ label }}</div>
+    <div :class="['font-[600]', labelStyle]">{{ label }}</div>
```

Comment:
Maybe create a new component `<NFTBookSpecTableValue />` for this div

Reply:
[💄 Add <NFTBookSpecTableValue/>](https://github.com/likecoin/liker-land/pull/1883/commits/fea2e8c7df386da746997b8125d100f6751584ae)